src/main/java/page_objects/LoginPage.java

```diff
@@ -0,0 +1,45 @@
+package page_objects;
+
+import common.helpers.BrowserHelper;
+import common.helpers.ElementHelper;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+    //Locators
+    private final By txtEmail = By.id("username");
+    private final By txtPassword = By.id("password");
+    private final By btnLogin = By.xpath("//input[@title='Login']");
+    private final By lblErrorMsg = By.xpath("//p[@class='message error LoginForm']");
```

Comment:
can we use css selector with a single class here?

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.